daffodil-core/src/main/scala/org/apache/daffodil/dsom/GroupRef.scala

```diff
@@ -129,18 +131,15 @@ final class ChoiceGroupRef(
   refXML: Node,
   refLexicalParent: SchemaComponent,
   positionArg: Int,
-  isHiddenArg: Boolean)
+  override val isHidden: Boolean)
   extends ChoiceTermBase(refXML, Option(refLexicalParent), positionArg)
   with GroupRef {
 
   requiredEvaluationsIfActivated(groupDef)
 
   private lazy val globalGroupDef = globalGroupDefArg // once only
 
-  override def isHidden = isHiddenArg
-  override def isHiddenGroupRef = isHiddenArg
-
-  override lazy val groupDef = globalGroupDef
+  override lazy val groupDef = LV('cgrGroupDef){ globalGroupDef }.value
```

Comment:
Is it important for the symbol used in the LV match the variable name, maybe for debugging or diagnostic purposes?

Reply:
There's a comment under LV, where it says 2 symbols with the same name could cause issues that tend to be difficult to debug, so I named it something different since SGR's groupDef already has a groupDef LV. I also found some precedence where the symbol didn't match the variable name (such as in org/apache/daffodil/dsom/SchemaDocIncludesAndImportsMixin.scala:151) so I figured it'd be ok